### PR TITLE
[fix #5938] New Lean Data pages

### LINKED ISSUE #5938: [Oct 1] Lean Data Practices site launch
**Lean Data Practices Landing page + 4 supporint Pages:** 
[Project Board](https://github.com/mozilla/bedrock/projects/10)
[Master Project Doc](https://docs.google.com/document/d/1gLnXiXzR-gQJq04x0RRiOjeqsP6GCgGL34Xf6y2K1iQ/edit)
[Current Page](https://www.mozilla.org/en-US/about/policy/lean-data/)
Resources from Client
[Copy doc](https://docs.google.com/document/d/1NBKCDZt8PJjPXP1yHNOozKx7ChUMYfAMIreTNpURH0g/edit)

**Success Criteria**
- [ ] Live on October 1, 2018
(The Policy team is doing presentations in the fall and has asked for a target date for go-live of October 1)
- [x] Maintain current URL redirect from leandatapractices.com to https://www.mozilla.org/en-US/about/policy/lean-data/
- [ ] Design and Build Landing Page
- [ ] Design and Build Stay Lean Page (this should be the template for the following pages)
- [ ] Design and Build Security Page 
- [ ] Design and Build Engage Users Page 
- [ ] Design and Resources Page

### PR DESCRIPTION
**Pull request by craigcook**
## Description
New lean data pages.

## Issue / Bugzilla link
#5938 
#6099 

## Testing
https://bedrock-demo-craigcook.oregon-b.moz.works/about/policy/lean-data/

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/about/policy/lean-data/engage-users.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  </div>
+
+  <div class="mzp-c-hero-image">
+    {{ high_res_img('mozorg/about/lean-data/engage-users-hero.png', {'alt': '', 'width': '424', 'height': '424'}) }}
+  </div>
+</section>
+
+<section>
+  <div class="mzp-l-content l-content-narrow">
+
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/policy/lean-data/engage-users.html]: Nit remove extra blank space

File: bedrock/mozorg/templates/mozorg/about/policy/lean-data/engage-users.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      <li>{{ _('<a href="%s">A Strong Privacy Policy can Save Your Company Millions</a> by Harvard Business Review.'|format('https://hbr.org/2018/02/research-a-strong-privacy-policy-can-save-your-company-millions')) }}</li>
+      <li>{{ _('<a href="%s">Building a privacy policy people actually want to read</a>, on Medium.'|format('https://medium.com/juro-blog/privacy-by-design-building-a-privacy-policy-people-actually-want-to-read-bd8eb4a52403')) }}</li>
+      <li>{{ _('<a href="%s">Transparency Reporting Index</a> by Access Now.'|format('https://www.accessnow.org/transparency-reporting-index/')) }}</li>
+      <li>{{ _('<a href="%s">Transparency Reporting Toolkit</a> by New America’s Open Technology Institute and Harvard University’s Berkman Center for Internet & Society.'|format('https://www.newamerica.org/oti/policy-papers/the-transparency-reporting-toolkit/')) }}</li>
+      <li><a href="https://docs.google.com/document/d/1TJafYmeFT3LIQe6YPLEHWwh37PGbK2RunntDIaN2lL8/">{{ _('Download the full toolkit.') }}</a></li>
+    </ul>
+
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/policy/lean-data/engage-users.html]: Nit remove extra blank space

File: bedrock/mozorg/urls.py [hunk 3]
```diff
@@ -15,7 +15,10 @@
     page('about/manifesto', 'mozorg/about/manifesto.html'),
     page('about/manifesto/details', 'mozorg/about/manifesto-details.html'),
     page('about/leadership', 'mozorg/about/leadership.html'),
-    page('about/policy/lean-data', 'mozorg/about/policy/lean-data.html'),
+    page('about/policy/lean-data', 'mozorg/about/policy/lean-data/index.html'),
```
Comment (alexgibson) [inline bedrock/mozorg/urls.py]: I believe this breaks a few URL reverse lookups for `url('mozorg.about.policy.lean-data')`, so you may need to update on pages that link to the lean data landing page.

Comment (craigcook) [reply bedrock/mozorg/urls.py]: Oh good call, knew I'd forget some details in the rush...

**Pull request merged**